DevCycle.SDK.Server.Local/ConfigManager/EnvironmentConfigManager.cs

```diff
@@ -139,6 +150,13 @@ private async Task FetchConfigAsyncWithTask()
                     logger.LogError("Error loading config. Using cache etag: {ConfigEtag}. Exception: {Exception}", configEtag, e.Message);
                 }
             }
+            finally
+            {
+                if (!Initialized)
+                {
+                    OnInitialized(dvcEventArgs);
```

Comment:
Any reason we're only calling this on failure to init instead of always? SetConfig doesn't call the oninit to pass the success back to the event subscriber

Reply:
My sick brain wrote the test wrong which gave me false confidence. Resolved.